FFCDemoPaymentService/obj/.snyk

```diff
@@ -5,41 +5,41 @@ ignore:
   SNYK-DOTNET-SYSTEMTEXTENCODINGSWEB-1253267:
```

Comment:
How'd you get this to run?

Also, are these issues still at large -- if you empty this file and then run `snyk` command, does it populate to this?

Reply:
`snyk test` worked for me.
I believe they are all still active as this was blocking the pipeline build for a while and I checked the snyk portal which highlighted all of these, the only i could get round it was to update the expiry date.
John explained that because its dotnet some of the packages have vulnernabilities that have no fix so we have to just accept them and add it to the snyk exclude.